### ISSUE TITLE
CLA assistant not registering my signature

### ISSUE THREAD

**Issue by dougollerenshaw** [cla-assistant/cla-assistant#563]
I'm having trouble with the CLA assistant on my organization's github page. After following the process to sign, I'm still not listed as signing. The issue has persisted for over a week despite multiple attempts to sign.

One of the repo admins asked me to log an issue here (https://github.com/AllenInstitute/AllenSDK/pull/1474#issuecomment-620747779)

URL to the linked Repo/Org:
https://github.com/AllenInstitute/AllenSDK
https://github.com/AllenInstitute/AllenSDK/pull/1474
https://github.com/AllenInstitute/AllenSDK/pull/1474#issuecomment-620747779
Steps to reproduce the problem:

1. I click the "CLA not signed yet" button
2. I get routed to the "Please sign the CLA" page
3. I click "sign in with github to agree"

What is the expected result?

I expect the CLA to show up as 'signed' for my username

What happens instead?

There is still a red X next to my username in the CLAassistant box on github. However, there is additional text that reads: "You have signed the CLA already but the status is still pending? Let us recheck it."
![image](https://user-images.githubusercontent.com/19944442/80519533-fcc47d00-893c-11ea-8b71-57e52a764adf.png)

**Comment by dougollerenshaw** [comment]
I solved this issue. It turned out that I had a long-forgotten github account associated with one of my email addresses. For reasons I don't fully understand, github was associating my commits with that other username, even though I was logged in under my primary username when making those commits.

Presumably, CLA Assistant was having trouble associating the username with which I was signed in when signing the agreement with the username that was (erroneously) associated with my contributions. 

I signed in under the alternate github ID to sign the CLA, which solved the immediate issue. I also cleared up the github username/email conflict to avoid future problems.
